### DISCUSSION TITLE
Fatal Error When Using Ray as MU Plugin in WordPress

### DISCUSSION THREAD

**Discussion by doroness** [spatie/wordpress-ray#66] [Q&A]
When trying to install the Ray plugin using the MU method (as specified [here](https://spatie.be/docs/ray/v1/installation-in-your-project/wordpress))

I'm getting an error: `Fatal Error...Class "Spatie\WordPressRay\Ray" not found`.

![image](https://user-images.githubusercontent.com/53006624/216900329-a6502573-3550-449b-89b4-0f8dba016ad3.png)

Here is a snapshot of my editor and file structure

![image](https://user-images.githubusercontent.com/53006624/216900974-b620dffd-2884-46e0-a5d0-295957a59eab.png)

Why? What am I missing?

Thanks

**Comment by jeffreyvr** [comment]
Hi @doroness, from the `wordpress-ray` directory run:

```bash
composer install -o --no-dev
```

Error should be resolved then. I've created a [PR](https://github.com/spatie/ray/pull/775) to make mention of this in the docs.

**Comment by doroness** [comment]
Hi @jeffreyvr  
Thanks - it worked :)

Can you explain briefly what the issue?

Currently, I'm working g on a local dev environment what should I do when the site is uploaded to a remote server?

**Comment by jeffreyvr** [comment]
Happy it worked! The vendor directory is no longer in git. This means that when you clone the project from GitHub, you miss the vendor directory and running the composer command fixes that.

Alternatively, you can also download the complete plugin in ZIP-format on the [releases page](https://github.com/spatie/wordpress-ray/releases) (`spatie-ray.zip`) instead of cloning it from GitHub (these include the vendor folder).
